src/System.Text.RegularExpressions/ref/System.Text.RegularExpressions.cs

```diff
@@ -197,7 +200,7 @@ public partial class Regex : System.Runtime.Serialization.ISerializable
         public override string ToString() { throw null; }
         public static string Unescape(string str) { throw null; }
         protected bool UseOptionC() { throw null; }
-        protected bool UseOptionR() { throw null; }
+        protected internal bool UseOptionR() { throw null; }
```

Comment:
Not sure why this got changed. Sources out of sync?

Reply:
protected internal means it's accessible to protected or internal, but internal isn't relevant to a ref assembly, so it was previously dropped apparently (and dropping it has no impact on consumers using the ref assembly). It doesn't hurt to have it if the tool is generating it, though I'm a little surprised the tool is generating it.